src/test/java/core/basesyntax/FileWriter.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriter {
+    public static void writeDataToFile(String filePath, String data) {
+        File file = new File(filePath);
+        if (file.exists()) {
+            try {
+                file.createNewFile();
+            } catch (IOException e) {
+                throw new RuntimeException("Can't create file " + filePath, e);
+            }
+        }
+        try (BufferedWriter bufferedWriter = new BufferedWriter(
+                new java.io.FileWriter(filePath, false))) {
+            bufferedWriter.write(data);
```

Comment:
а чому якщо file.exists() == true ми викликаємо file.createNewFile ?
```suggestion
        try (BufferedWriter bufferedWriter = new BufferedWriter(
                new java.io.FileWriter(filePath, false))) {
            bufferedWriter.write(data);
```

Reply:
Оу, там должно было быть !file.exists()